src/server.rs

```diff
@@ -23,6 +23,12 @@ pub enum Error {
     Accept(std::io::Error),
 }
 
+#[derive(Debug)]
+pub enum ListenerType {
```

Comment:
@Raelr Did you plan to start using this enum? Or can it be removed since it's still unused?